packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -263,6 +263,26 @@ void main() {
         expect(scrollView.controller, primary2);
       });
 
+      testWidgets('Use custom ScrollController when scrollController is set', (WidgetTester tester) async {
+        final ScrollController customController = ScrollController();
+        final Widget reorderableList = ReorderableListView(
+          scrollController: customController,
+          children: const <Widget>[
+            SizedBox(width: 100.0, height: 100.0, child: Text('C'), key: Key('C')),
+            SizedBox(width: 100.0, height: 100.0, child: Text('B'), key: Key('B')),
+            SizedBox(width: 100.0, height: 100.0, child: Text('A'), key: Key('A')),
+          ],
+          onReorder: (int oldIndex, int newIndex) { },
+        );
+
+        await tester.pumpWidget(MaterialApp(home: reorderableList));
+        final ReorderableListView listView = tester.widget(
+          find.byType(ReorderableListView),
+        );
+
+        expect(listView.scrollController, customController);
```

Comment:
Instead of checking if `listView.scrollController == customController`, it's better to perform some operation on `customController` and see if that behavior correctly reflects on the UI. 

Let me know if you need any help with this!

Reply:
@shihaohong yeah what would you recommend here?

Reply:
For example, passing in your own ScrollController and making sure that manipulating it results in visual changes. And also, when making interactions with the ReorderableListView (like dragging it around), the ScrollController's values are properly updated.

Reply:
I'll give it a shot, I think there's an example in the existing test file for it that I could base it off.

Reply:
`tests/material/tabs_test.dart` should also have some tests that use `controller.jumpTo()`, then you could check the offsets of the children of the list view and see if they're expected

Reply:
I added a jumpTo test (docs say this has no animations so I didn't call pump after setting jumpTo), but the tabs_test is for the TabController tests. Unrelated to this issue.